src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  padding: 10px;
+  width: $calendar-width;
```

Comment:
The variable `$calendar-width` is used here, but it's not defined within this file. Make sure this variable is defined elsewhere in your SCSS files to prevent compilation issues.